Gemfile

```diff
@@ -63,6 +63,7 @@ group(:development, :test) do
   gem 'webmock', '~> 1.24'
   gem 'vcr', '~> 2.9'
   gem "hocon", :require => false
+  gem "hiera-eyaml", :require => false
```

Comment:
Or not. This is just for testing. It would be very nice if the tests fail when a new version comes out that isn't compatible.